pytorch_lightning/lite/lite.py

```diff
@@ -208,7 +213,7 @@ def setup_dataloaders(
 
     def _setup_dataloader(
         self, dataloader: DataLoader, replace_sampler: bool = True, move_to_device: bool = True
-    ) -> Union[Iterable, DataLoader]:
+    ) -> Iterable:
```

Comment:
`isinstance(DataLoader, Iterable)` returns False so I'm not sure these typing changes are correct.

Reply:
hopefully addressed in  #10625